### ISSUE TITLE
more instruction for labeling tool

### ISSUE THREAD

**Issue by mindmad** [liulei01/DRBox#40]
Hello 

If you don't mind giving more instruction for labeling tool.
when I use it the .rbox file stays empty.
and also when labeling there is no feedback I mean no line or anything that shows it creating a label

**Comment by smajida** [comment]
@mindmad you can send your data to TernowAI www.ternowai.eu to be annotated

**Comment by mindmad** [comment]
@smajida  thank you for your support 
However, as I have read their annotated structure is not the same as this repository require . and also I don't think the annotated for free.

**Comment by arasharchor** [comment]
@mindmad yeah, but Ternow provides very inexpensive and very fast labeling
